cmake/kokkos_arch.cmake

```diff
@@ -375,6 +376,18 @@ IF (KOKKOS_ARCH_HSW)
   )
 ENDIF()
 
+IF (KOKKOS_ARCH_RISCV_MILKV)
+  COMPILER_SPECIFIC_FLAGS(
+    COMPILER_ID KOKKOS_CXX_HOST_COMPILER_ID
+        Cray    NO-VALUE-SPECIFIED
+        Intel   NO-VALUE-SPECIFIED
+        MSVC    NO-VALUE-SPECIFIED
+        NVHPC   NO-VALUE-SPECIFIED
```

Comment:
```suggestion
```
Lets just delete the other ones: the general path here is that we say what it should be by default and add special values if we know it works with a compiler by providing something else than the default.